docs/fitting_types.rst

```diff
@@ -385,6 +385,33 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting multidimensional datasets
+--------------------------------
+Imagine that you have now a specific property over a grid, like the surface height,
+so you have a three-dimensional data set. To fit a function with two parameters
+to this data set, you can use :mod:`symfit`.
+
+Let's work this out with the following example. We made a polynomial function
+with two coefficients, representing two terms of mixed order in ``x`` and ``y``
+
+:math:`f(x, y) = z = c2 \cdot x^4 \cdot y^5 + c1 \cdot x \cdot y^2`
+
+First, we have to define our model::
+
+  x, y, z = variables('x, y, z')
+  c1, c2 = parameters('c1, c2')
+  model_dict = {z: Poly( {(1, 2): c1, (4, 5): c2}, x ,y)}
+  model = Model(model_dict)
+  #printed version: z(x, y; c1, c2) = Poly(c2*x**4*y**5 + c1*x*y**2, x, y, domain='ZZ[c1,c2]')
+
+Now I can fit this polynomial model to some data::
+
+  fit = Fit(model, x=xdata, y=ydata, z=zdata)
+  fit_result = fit.execute()
+
+In conclusion, we made a polynomial function with two parameters,
+which coordinates of a mesh could be.
```

Comment:
Actually it is a function of both two parameters and two variables. The variables span the mesh (x, y), and we obtained two parameters.

Reply:
Actually we should probably also comment that xdata and ydata here are two dimensional, not one dimensional.

I think you should actually generate some mock data here using `np.meshgrid` to really make this example clear.

Reply:
> Actually it is a function of both two parameters and two variables. The variables span the mesh (x, y), and we obtained two parameters.

I improved the text bevor I read your comment. Fits that as it is now?

Reply:
> Actually we should probably also comment that xdata and ydata here are two dimensional, not one dimensional.
> 
> I think you should actually generate some mock data here using `np.meshgrid` to really make this example clear.

Just to be clear: xdata, ydata and zdata are two dimensional, right? :D

Reply:
yes indeed, all have the same shape :).

Reply:
I would add something like this to the second code block:

`x = np.arange(-10, 10, 0.4)`
 `y = np.arange(-10, 10, 0.4)`
 `xdata, ydata = np.meshgrid(x, y)`
 `zdata = 42 * xdata**4 * ydata**5 + 3.14 * xdata * ydata**2`

Reply:
Yeah, I like that, especially your choice of parameter values ;).

Small change: with non integer steps it is better to use `np.linspace`.